### DISCUSSION TITLE
New to Okta and OWASP; how to setup?

### DISCUSSION THREAD

**Discussion by wbb3834** [DependencyTrack/dependency-track#2155] [Q&A]
Just got OWASP DT running, anyone set this up w/ Okta yet?
How do I find the sign in/out redirect URI's Okta is asking for?
Thanks.

**Comment by syalioune** [comment]
Hello @wbb3834 
The redirect URI is most likely https://YOUR_DT_FQDN/static/oidc-callback.html

**Reply by wbb3834** [reply]
@syalioune Thanks for reaching out, is this URI the same for both signin/out?

**Reply by syalioune** [reply]
That's definitively the URI for signin.
At the moment, DT does not support OIDC logout. See #2015 && #2016
